src/chainstate/coordinator/tests.rs

```diff
@@ -3855,3 +3855,78 @@ fn preprocess_block(
         )
         .unwrap();
 }
+
+#[test]
+fn test_check_chainstate_db_versions() {
+    let path = "/tmp/stacks-blockchain-check_chainstate_db_versions";
+    let _ = std::fs::remove_dir_all(path);
+
+    let sortdb_path = format!("{}/sortdb", &path);
+    let chainstate_path = format!("{}/chainstate", &path);
+
+    let epoch_2 = StacksEpoch {
+        epoch_id: StacksEpochId::Epoch20,
+        start_height: 0,
+        end_height: 10000,
+        block_limit: BLOCK_LIMIT_MAINNET_20.clone(),
+        network_epoch: PEER_VERSION_EPOCH_2_0,
+    };
+    let epoch_2_05 = StacksEpoch {
+        epoch_id: StacksEpochId::Epoch2_05,
+        start_height: 0,
+        end_height: 10000,
+        block_limit: BLOCK_LIMIT_MAINNET_205.clone(),
+        network_epoch: PEER_VERSION_EPOCH_2_05,
+    };
+
+    // should work just fine in epoch 2 if the DBs don't exist
+    assert!(
+        check_chainstate_db_versions(&[epoch_2.clone()], &sortdb_path, &chainstate_path).unwrap()
+    );
+
+    // should work just fine in epoch 2.05 if the DBs don't exist
+    assert!(
+        check_chainstate_db_versions(&[epoch_2_05.clone()], &sortdb_path, &chainstate_path)
+            .unwrap()
+    );
+
+    StacksChainState::make_chainstate_dirs(&chainstate_path).unwrap();
+
+    let sortdb_v1 =
+        SortitionDB::connect_v1(&sortdb_path, 100, &BurnchainHeaderHash([0x00; 32]), 0, true)
+            .unwrap();
+    let chainstate_v1 = StacksChainState::open_db_without_migrations(
+        false,
+        0x80000000,
```

Comment:
nit: Could use the constant for this `CHAIN_ID_TESTNET`.

Reply:
Done